src/js/mobile.js

```diff
@@ -124,7 +124,6 @@
 			"./core/router/Router",
 			"./core/router/route/page",
 			"./core/router/route/popup",
-			"./core/router/route/popup",
```

Comment:
Why do you remove the code?
Isn't it needed?

Reply:
This is a duplicate of the code just above.